packages/fxa-content-server/app/scripts/lib/auth-errors.js

```diff
@@ -258,7 +258,7 @@ var ERRORS = {
     errno: 202,
     message: t('Feature not enabled')
   },
-  BACKEND_SERVICE_FAILURE: {
+  DISABLED_CLIENT_ID: {
```

Comment:
Is it intended to remove the backend service failure error? Also disabled client is 204 on auth-server [errors](https://github.com/mozilla/fxa/blob/9360599f81f2b7841c4ebcff18ee593ee6feb90c/packages/fxa-auth-server/lib/error.js#L98).

Reply:
This must have been some sort of copy-paste fail, nice catch.